### PR TITLE
Replace master with main

### PR DESCRIPTION
**Pull request by mkelley**
Update readme with instructions on how to locally rename master to main.  String replace relevant "master" text with "main".

Addresses #274

### CODE REVIEW

File: website/public/index.html [hunk 1]
```diff
@@ -1,130 +1,134 @@
...
+    <section id="hero">
+      <img src="images/sbpy_logo_full.png" width="524"
+        onerror="this.src='images/astropy_project_logo.png'; this.onerror=null;"
+        alt="sbpy - A Python Module for Small-Body Planetary Astronomy" />
+    </section>
+
+    <section class="whatsnew">
+      <p>sbpy is an <a href="https://astropy.org">Astropy</a> affiliated package for asteroid and comet researchers.</p>
+      <a class="reference external image-reference" href="https://travis-ci.org/NASA-Planetary-Science/sbpy">
+        <img alt="Travis-CI status" src="https://travis-ci.org/NASA-Planetary-Science/sbpy.svg?branch=master" /></a>
```
Comment (migueldvb) [inline website/public/index.html:64]: This is a reference to master that was not replaced apparently ^^^

**Comment by mkelley** [comment]
Remove instructions about renaming master to main, github provides an excellent pop-up with the right commands.

**Comment by mkelley** [comment]
Hi @jianyangli , is this OK to merge?  It shouldn't interfere with our existing PRs .

**Review by jianyangli — APPROVED** [review]
Yes, I think it's ready to merge.  I don't see any problems.

**Pull request merged**